Update BigQuery Source info re Service Account Key (#908)
Amplitude-Developer-Docs/amplitude-dev-center: docs/data/sources/bigquery.md

Each group property in `group_properties` would be applied to every group in `groups`

> ## Updating Your BigQuery Service Account Key

Reviewer: ```suggestion
## Update your BigQuery service account key
```

Sentence case, and Bigquery does not capitalize "service account key"

> In order for you to update the [Service Account](https://cloud.google.com/iam/docs/service-account-overview) used for your BigQuery Source, you must select the existing BigQuery Source within the Sources section of Data and select the ⚙️ icon. Within the modal, you can upload the new Service Account Key you want Amplitude to use moving forward. 

Reviewer: ```suggestion
To update the [Service Account](https://cloud.google.com/iam/docs/service-account-overview) used for your BigQuery Source, select the existing BigQuery Source within the Sources section of Data and click the ⚙️ (gear) icon. Within the modal, upload the new Service Account Key you want Amplitude to use moving forward. 
```

Removing some extra words to improve readability.

!!!warning "Service Account Data Access"

>       Before updating your Service Account Key, ensure the new Service Account Key has the proper data access to ensure Amplitude can successfully import any relevant data. 

Reviewer: ```suggestion
      Before you update your Service Account Key, ensure the new Service Account Key has the proper data access to ensure Amplitude can successfully import any relevant data. 
```

>       Before updating your Service Account Key, ensure the new Service Account Key has the proper data access to ensure Amplitude can successfully import any relevant data. 

Reviewer 2: @bh202205 do we auto perform permission checks when service account being updated? Would like to not rely on customer to do this type of manual checks which is not safe to us && effort consuming to customer.

Reviewer 3: No, we don't currently have the permission checks when the service account key being updated. It will perform some simple checks such as if it is a JSON file, but not verifying the access like what we do during the setup. It was actually due to the components we use:  `SetupFileInput` vs `FalconVerify`. I agree with you but I am not sure if there is a quick/easy way we can achieve this. 

Hi @ajhorst I was wondering if you could share more insights or ideas here. This is the related PR: https://github.com/amplitude/javascript/pull/30373
